Update hike.md (#2557)
writethedocs/www: docs/conf/portland/2026/hike.md

Every year we have a conference hike that kicks off the **unofficial** beginning of the conference. It has become a fantastic tradition and is a wonderful way to gather the day before the conference begins.

This year, we're offering two options to better accomodate interests. There is a 5-mile nature hike and shorter urban walk that explores the International Rose Test Garden and ends at Powell's Books. 

## Nature Hike
We will be hiking in the amazing [Forest Park](https://forestparkconservancy.org/), the largest urban forested park in the country. It is conveniently located in Northwest Portland, and has a beautiful walk along a creek. 

**The hike is around 5 miles long with 1,200 feet of elevation gain and is classified as a moderate hike. We’ll be going nice and slow so people can appreciate the views and forest.**

It's rained on us in the past, but we have faith it will be beautiful this year! We will hopefully see Mount Hood at the top :)

### Hike Schedule
- **Date: {{ hike.date }}**
- **Arrival:** Meet 15 minutes before the start time.

> - **Lunch (optional):** Join us at the [Nob Hill food carts](https://www.google.com/maps/place/Nob+Hill+Food+Carts/@45.5360531,-122.7007924,19.6z/data=!4m7!3m6!1s0x54950942eb34ca71:0xe277fed8c0cec152!8m2!3d45.5362156!4d-122.7000932!15sChZmb29kIGNhcnRzIG53IHBvcnRsYW5kkgEKZm9vZF9jb3VydOABAA!16s%2Fg%2F11vwhg4f9_?entry=tts) at 12:30pm for lunch. We'll leave for the trailhead at 1:40pm.

Reviewer: Should reverse these two probably

Every year we have a conference hike that kicks off the **unofficial** beginning of the conference. It has become a fantastic tradition and is a wonderful way to gather the day before the conference begins.

> This year, we're offering two options to better accommodate interests. There is a 5-mile nature hike and shorter urban walk that explores the International Rose Test Garden and ends at Powell's Books. 

Reviewer: ```suggestion
This year, we're offering two options to better accommodate interests:

 - a 5-mile uphill nature hike exploring Forest Park and Hoyt Arboretum
 - a shorter urban walk that explores the International Rose Test Garden and ends at Powell's Books
```

Every year we have a conference hike that kicks off the **unofficial** beginning of the conference. It has become a fantastic tradition and is a wonderful way to gather the day before the conference begins.

This year, we're offering two options to better accommodate interests:

>  - a 5-mile uphill nature hike exploring Forest Park and Hoyt Arboretum

Reviewer 2: ```suggestion
 - a 5-mile uphill hike exploring Forest Park and Hoyt Arboretum
```

 - a shorter urban walk that explores the International Rose Test Garden and ends at Powell's Books

> ## Nature Hike

Reviewer 2: Nature Hike is weird phrasing?

Reviewer 2: ```suggestion
## Hike
```

- **Lightweight rain jacket** - it likely will not be too cold, but it might drizzle on you.
- **High spirits!**

## Tickets

> Participating in the hike or urban walk is free, but please [register for your ticket]({{ hike.register_url }}) so that we can confirm number of folks on the rose garden tour and contact you in advance for day-of weather and logistics information. 

Reviewer: We should probably split these up? Or better have a checkbox in the registristration.

Reviewer 2: Yea, we need different URLs yea?

Reviewer: Or a radio box specifying which one you're going to in Tito? Might be less hassle?

Reviewer 2: Yea, we could also do that on the ticket itself with activities, but people always find that UX confusing.
